GVFS/GVFS/CommandLine/PrefetchVerb.cs

```diff
@@ -226,32 +264,75 @@ private void PrefetchCommits(ITracer tracer, GVFSEnlistment enlistment, GitObjec
             }
         }
 
-        private void PrefetchBlobs(ITracer tracer, GVFSEnlistment enlistment, GitObjectsHttpRequestor blobRequestor, CacheServerInfo cacheServer)
+        private void LoadBlobPrefetchArgs(
+            ITracer tracer,
+            GVFSEnlistment enlistment,
+            out string headCommitId,
+            out List<string> filesList,
+            out List<string> foldersList,
+            out FileBasedDictionary<string, string> lastPrefetchArgs)
         {
-            BlobPrefetcher blobPrefetcher = new BlobPrefetcher(
-                tracer,
-                enlistment,
-                blobRequestor,
-                ChunkSize,
-                SearchThreadCount,
-                DownloadThreadCount,
-                IndexThreadCount);
-
             string error;
-            if (!BlobPrefetcher.TryLoadFolderList(enlistment, this.Folders, this.FoldersListFile, blobPrefetcher.FolderList, out error))
+            
+            if (!FileBasedDictionary<string, string>.TryCreate(
+                    tracer,
+                    Path.Combine(enlistment.DotGVFSRoot, "LastBlobPrefetch.dat"),
+                    new PhysicalFileSystem(),
+                    out lastPrefetchArgs,
+                    out error))
+            {
+                tracer.RelatedWarning("Unable to warn last prefetch args: " + error);

```

Comment:
Is there a typo here?

`Unable to warn`

Should this say "Unable to load" (or something similar)?

Reply:
Typing faster than I can think apparently :-). I'll fix this up.